python/seldon_core/metrics.py

```diff
@@ -135,13 +135,35 @@ def update(self, custom_metrics: List[Dict], method: str):
                 value = worker_data.get(key, {}).get("value", 0)
                 worker_data[key] = {"value": value + metrics["value"], "tags": tags}
             elif metrics_type == "TIMER":
+                try:
+                    bins = json.loads(
+                        tags.get("__SELDON_TIMER_BINS__", json.dumps(BINS))
```

Comment:
It'd be cleaner to have this as a helper method defined at the top of the file, next to `split_image_tag` helper method.